lib/modules/database.js

```diff
@@ -232,7 +232,7 @@ class DatabaseRef extends ReferenceBase {
     const path = this.dbPath();
     return this.db.off(path, evt, origCB)
       .then(({callback, subscriptions}) => {
-        if (dbSubscriptions[path] && dbSubscriptions[path][evt].length > 0) {
+        if (subscriptions[path] && subscriptions[path][evt].length > 0) {
```

Comment:
I'm not sure what error you were seeing, but by changing this to `subscriptions`, it becomes functionally broken since we want to remove the JS callbacks from the global `dbSubscriptions` set. The `subscriptions` you referenced comes from: https://github.com/chaitanya0bhagvan/react-native-firestack/blob/483e15fd4b7a8256bec99771220c4cc9721e198a/lib/modules/database.js#L490. Probably needs a better name though!